backend/src/node.rs

```diff
@@ -226,7 +226,7 @@ impl Node {
         if let Ok(stringified) = serde_json::from_str::<String>(json) {
             Some(stringified.into())
         } else {
-            Some(json.into())
+            Some(json.to_owned().into())
```

Comment:
Some docs would be good on this method, explaining the old vs new node behaviour? How long is it reasonable to support old nodes?

Reply:
New version of `bytes` crate does not have conversion from non-static slices to `Bytes`, so need to clone the JSON dump here. This only happens on the manual network state requests, which happen rarely.